packages/api/src/graphql/article.ts

```diff
@@ -325,12 +325,21 @@ export const GraphQLPublicArticle: GraphQLObjectType<
     comments: {
```

Comment:
could we include the "CommentRatingSystem" so that the frontend knows, which CommentRatingSystemAnswers are available to vote on? Would save an extra query ;)

Reply:
You should be able to do that via GraphQL already, or do you want it attached to the comment?
Don't think there would be a clean way to do that there